### PR TITLE
Windows Onboarding

### PR DESCRIPTION
**Pull request by mdroidian**
Left out creating a user token and apps table seed

`token`
>We have a webhook on Clerk that assigns a user a token whenever a new one is created. That webhook did not hit your local endpoint bc it's set up to hit mine. I'll have to think through what a proper workaround here is (my first thought is to start setting up clerk in /mocks), but in the meantime, you could look at the code in /api/user/post.ts to see how to manually give yourself a token.

```
        getMysql().then((cxn) =>
          cxn
            .select({
              count: sql`COUNT(uuid)`,
            })
            .from(tokens)
            .where(eq(tokens.userId, userId))
            .then(async ([a]) =>
              !a?.count
                ? await cxn.insert(tokens).values({
                    uuid: v4(),
                    value: await randomString({
                      length: 12,
                      encoding: "base64",
                    }),
                    createdDate: new Date(),
                    userId,
                  })
                : Promise.resolve()
            )
            .then(() => cxn.end())
        );
```

`seed`
>So we're also going to need to seed your database with records in the apps table. I should create a script to make this easier. Try running:
```
INSERT INTO samepage_network.apps (id, code, name, workspace_label, live) VALUES (1, 'roam', 'Roam', 'graph', false)
```

### CODE REVIEW

**Comment by mdroidian** [comment]
@dvargas92495 take a look.

Although I see the check failed at `StripeAuthenticationError: You did not provide an API key.`

**Comment by dvargas92495** [comment]
Did you not have permission to request review? If not, I'll take a look at the settings.

You'll need to add STRIPE_SECRET_KEY as a github secret in your fork of the repository.

Getting the tests to pass on the PR won't be required yet since they're still a bit unstable, but it's an ideal we should shoot for

File: .env.default [hunk 1]
```diff
@@ -6,5 +6,8 @@ STRIPE_PUBLIC=pk_test_51HERaHFHEvC1s7vkxMfTOWuGCHWCyukcRbwF5WNOUgeWWkAUHj2btlzxG
 
 # Request the following values from support
 # CLERK_SECRET_KEY
-# STRIPE_SECRET
+# STRIPE_SECRET_KEY
 # STRIPE_WEBHOOK_SECRET
+
+# Uncomment the following line to enable debug logging
+# DEBUG=api,sendmessage
```
Comment (dvargas92495) [inline .env.default:13]: Let's actually move this to the contribution docs.

Mac users will be able to just do `DEBUG={keys} npm start`, and set of supported `{keys}` include more than just `api` and `sendmessage`

File: docs/developer/how_to_contribute.md [hunk 2]
```diff
@@ -34,9 +36,13 @@ The following directories also exist but are expected to be temporary:
...
-    1. `CLERK_API_KEY` - Get from `@dvargas92495`
-1. Ensure you have a local instance of `mysql` running, with a user with username and password both with the value `samepage_network`, running on port 3306, and a database created called `mysql`.
-1. Apply the SamePage schema to your local `mysql` instance by running `npx ts-node scripts/cli.ts plan --sql`, followed by `npx ts-node scripts/cli.ts apply --sql`.
+   1. `CLERK_SECRET_KEY` - Get from `@dvargas92495`
+   2. `STRIPE_SECRET_KEY` - Get from `@dvargas92495`
+1. Ensure you have a local instance of `mysql` running, with a user with username and password both with the value `samepage_network`, running on port 3306, and a database created called `samepage_network`.
+   1. Create Database: `CREATE DATABASE samepage_network;`
+   2. Create User: `CREATE USER 'samepage_network'@'localhost' IDENTIFIED BY 'samepage_network';`
+   3. Grant Privileges: `GRANT ALL PRIVILEGES ON samepage_network.* TO 'samepage_network'@'localhost';`
+1. Apply the SamePage schema to your local `mysql` instance by running `npx ts-node scripts/cli.ts plan --sql`, followed by `npx ts-node scripts/cli.ts apply --sql --bare`.
```
Comment (dvargas92495) [inline docs/developer/how_to_contribute.md:45]: Let's say `SamePage database schema`

There are tons of different schemas in the world of SamePage so it's good to clarify whenever we can

**Review by dvargas92495 — COMMENTED** [review]
Nice! Couple nits

**Comment by mdroidian** [comment]
> Did you not have permission to request review? If not, I'll take a look at the settings.

No dice on the permissions.

**Comment by mdroidian** [comment]
Hmm.  Added `STRIPE_SECRET_KEY` here, but test still failed.

image

Is that correct?

**Comment by dvargas92495** [comment]
Hmm that should be correct. I'll have to do some googling around to see if I'm missing something on how this works

**Pull request merged**